docs/getting-started/deploy.md

```diff
@@ -1,52 +1,17 @@
 ---
 id: deploy
-title: How to deploy your services
-sidebar_label: Deploy your services
+title: Deploy to Docker Desktop
+sidebar_label: Deploy to Docker Desktop
 slug: /deploy
 ---
 
-# Deploy your services
-
-Applications created with Amplication comprise two main components — the server that runs the API and a database.
-
-When it comes to deployment, you can choose one of the following options:
-
-- Downloading the generated source code and continuing the process with your favorite IDE and local tools.
```

Comment:
I think we should keep this section but instead of talking about "downloading" the code, we should talk about getting the code through GitHub

Reply:
We have a separate article for GitHub sync and I added a link to it.